modules/cnv_install_1_4.adoc

```diff
@@ -16,26 +16,26 @@ This procedure installs the following components:
 
 * A running {product-title} 3.11 cluster
 * User with *cluster-admin* privileges
-
-[NOTE]
-====
-If you execute the playbook from a remote host, that host must have the 
-{product-title} client and `ansible` package installed. The {ProductName} 
-repository must also be enabled.
-====
+* `*rhel-7-server-cnv-1.4-tech-preview-rpms*` must be enabled
 
 .Procedure
 
+. Install the `kubevirt-ansible` RPM and its dependencies:
++
+----
+$ yum install kubevirt-ansible
+----
+
 . Log into the {product-title} cluster as an admin user:
 +
 ----
 $ oc login -u system:admin
 ----
 
-. Install the `kubevirt-ansible` RPM and its dependencies:
+. Change directories to `/usr/share/ansible/kubevirt-ansible`:
 +
 ----
-$ yum install kubevirt-ansible
+$ cd /usr/share/ansible/kubevirt-ansible
```

Comment:
@simon3z explicitly asked not to leave the reader inside this directory. We can either use full paths as he requested, or use `pushd /usr/share/ansible/kubevirt-ansible` here, and `popd` in the end as I suggested.

Can we please get this doc converged? It feels sad to see @simon3z's comment done and then undone.

Reply:
As mentioned in another comment, here are the latest updates - the user is no longer left in `kubevirt-ansible`: https://github.com/openshift/openshift-docs/pull/13328/commits/6010dcf48c2bb4c3d411f0062cb7045bf28be1d2

Reply:
thanks! I still find it hard to review stuff on github, I'm used to a more linear review.

Reply:
No worries @dankenigsberg, it's extremely hard to keep track of what's being said, especially when force-pushing removes some comments. Thanks for the review!